nilearn/decoding/decoder.py

```diff
@@ -57,9 +55,9 @@
     ridge_regressor=RidgeCV(),
     ridge=RidgeCV(),
     svr=SVR(kernel='linear', max_iter=1e4),
-    dummy_classifier = DummyClassifier(strategy='prior',
-                                       random_state=0),
-    dummy_regressor = DummyRegressor(strategy='mean'),
+    dummy_classifier=DummyClassifier(strategy='stratified',
```

Comment:
Could you add spaces around "="?

Reply:
Sorry this is automatically corrected by my editor. Actually it's a bit weird because the above estimator definition does not have spaces around "=".  WDYT?

Reply:
Sorry, I misunderstood what you were saying @tbng, thanks for clarifying! 
Indeed, this convention is mostly respected across nilearn, but there are still a few exceptions here and there...